app/src/main/java/com/github/se/cyrcle/ui/zone/ZoneManagerScreen.kt

```diff
@@ -0,0 +1,53 @@
+package com.github.se.cyrcle.ui.zone
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Add
+import androidx.compose.material3.Scaffold
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.scale
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import com.github.se.cyrcle.R
+import com.github.se.cyrcle.ui.navigation.NavigationActions
+import com.github.se.cyrcle.ui.navigation.Screen
+import com.github.se.cyrcle.ui.theme.ColorLevel
+import com.github.se.cyrcle.ui.theme.atoms.IconButton
+import com.github.se.cyrcle.ui.theme.atoms.Text
+import com.github.se.cyrcle.ui.theme.molecules.TopAppBar
+
+/** Screen where users can manage their downloaded zones. and access the zone selection screen. */
+@Composable
+fun ZoneManagerScreen(navigationActions: NavigationActions) {
+  Scaffold(
+      topBar = {
+        TopAppBar(
+            title = stringResource(R.string.zone_manager_screen_title),
+            navigationActions = navigationActions)
+      }) { padding ->
+        Box(modifier = Modifier.padding(padding).fillMaxSize()) {
+          AddZoneButton(navigationActions)
```

Comment:
It's not that important, but here it would make it much easier if you just used the "floating action button" parameter of the scaffold rather than doing it like this. You wouldn't even have to worry about using a box and paddings and stuff.